geomstats/learning/radial_kernel_functions.py

```diff
@@ -0,0 +1,385 @@
+"""Radial kernel functions.
+
+References
+----------
+https://en.wikipedia.org/wiki/Kernel_(statistics)
+https://en.wikipedia.org/wiki/Radial_basis_function
+"""
+
+import geomstats.backend as gs
+
+
+def uniform_radial_kernel(distance, bandwidth=1.0):
+    """Uniform radial kernel.
+
+    Parameters
+    ----------
+    distance : array
+        Array of non-negative real values.
+    bandwidth : float, optional (default=1.0)
+        Positive scale parameter of the kernel.
+
+    Returns
+    -------
+    weight : array
+        Array of non-negative real values of the same shape than
+        parameter 'distance'.
+    """
+    if gs.any(distance < 0):
+        raise ValueError('The distance should be a non-negative real number.')
```

Comment:
The lines with the ValueErrors are not tested. Could you add the corresponding tests?

Testing that an error is raised is a test that looks like this:

https://github.com/geomstats/geomstats/blob/1e33eb3841cda03348a9640e247f86a244edaf75/tests/test_manifold.py#L20

Reply:
Also, these 4 lines occur several times in this file. Maybe factorize them into an auxiliary function `_check_distance_and_bandwidth`?